wgpu/_compute.py

```diff
@@ -0,0 +1,152 @@
+"""
+Simple high-level utilities for doing compute on the GPU.
+
+The purpose of wgpu-py is to provide a Pythonic wrapper around wgpu-native.
+In principal, a higher-level API is not within the scope of the project.
+However, by providing a few un-opinionated utility functions, other projects
+(like python-shader) can use wgpu (e.g. in their tests) without having
+to keep track of changes in wgpu itself.
+"""
```

Comment:
This is something that needs thought. How far should we go here? I feel pretty confident about adding this function, since it is very "self contained", you give it some arrays and what you expect as output, it runs the shader, and returns the buffer contents. But should we have something similar for textures? Or support rendering something to an offscreen texture?

Reply:
I think that wgpu-py should be focused on a small core, we can provide some utilities in an optional "util" module or subpackage to keep import times fast, but also let's make sure we don't end up building half of visvis2 here. :)

Reply:
> but also let's make sure we don't end up building half of visvis2 here. :)

exactly